readthedocs/core/tests/test_adapters.py

```diff
@@ -0,0 +1,174 @@
+from unittest import mock
+
+from allauth.exceptions import ImmediateHttpResponse
+from allauth.socialaccount.adapter import get_adapter as get_social_account_adapter
+from allauth.socialaccount.models import SocialAccount, SocialLogin
+from allauth.socialaccount.providers.github.provider import GitHubProvider
+from django.contrib.auth.models import AnonymousUser, User
+from django.test import TestCase
+from django_dynamic_fixture import get
+
+from readthedocs.allauth.providers.githubapp.provider import GitHubAppProvider
+
+
+class SAMLAdapterTest(TestCase):
```

Comment:
These aren't SAML tests right? 

```suggestion
class GitHubAppAdapterTest(TestCase):
```

Reply:
somehow I copied this from somewhere :_